### PR TITLE
Update release notes for CLI 2.0.10

### PR DESCRIPTION
**Pull request by derekbekoe**

### CODE REVIEW

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/cli-release-77/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?branch=pr-en-us-422) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C7%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201707071717042174-422%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 1]
```diff
@@ -16,6 +16,251 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+* Adding table output formatter for `az lab vm list` & `az lab vm show`
+
+### Monitor
+
+* Include autoscale template file to fix `az monitor autoscale-settings get-parameters-template` command (#3349)
+* BC: `monitor alert-rule-incidents list` renamed `monitor alert list-incidents`
+* BC: `monitor alert-rule-incidents show` renamed `monitor alert show-incident`
+* BC: `monitor metric-defintions list` renamed `monitor metrics list-definitions`
+* BC: `monitor alert-rules` renamed `monitor alert`
+* BC: `monitor alert create` completely revamped. `condition` and `action` no longer accepts JSON.
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: Should be split into several notes.

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 2]
```diff
@@ -16,6 +16,251 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+* `application-gateway http-listener create`: Default frontend IP if only one exists.
+* `application-gateway rule create`: Default address pool, HTTP settings, and HTTP listener if
+   only one exists.
+* `lb rule create`: Default frontend IP and backend pool if only one exists.
+* `lb inbound-nat-rule create`: Default frontend IP if only one exists.
+
+### Profile
+
+* account show: support output in SDK auth file format
+* Output deprecating information on using '--expanded-view'
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: Formatting(?)  Leader should be `Output:`

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 3]
```diff
@@ -16,6 +16,251 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+* Add get-access-token command to provide raw AAD token
+* Support login with a user account with no associated subscriptions
+
+### RDBMS
+
+* Add support for list server across a subscription.
+* [RDBMS]Support list servers across a subscription (#3417)
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: Duplicate of the above.

**Comment by derekbekoe** [comment]
I am closing the PR as it's from several weeks ago and we are releasing again soon.

**Pull request closed without merging**